exporters/trace/jaeger/uploader.go

```diff
@@ -41,6 +41,9 @@ type EndpointOption func() (batchUploader, error)
 // For example, localhost:6831.
 func WithAgentEndpoint(agentEndpoint string, options ...AgentEndpointOption) EndpointOption {
 	return func() (batchUploader, error) {
+		// Overwrite agent endpoint if environment variables are available.
+		AgentEndpointFromEnv(&agentEndpoint)
```

Comment:
This does not use [the specified defaults](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.0.1/specification/sdk-environment-variables.md#jaeger-exporter) if nothing is specified. We should updated to do this.